CONTRIBUTING.md

```diff
@@ -13,6 +13,19 @@ To get started with the project, run `yarn install` in the root directory to ins
 yarn install
 ```
 
+### Project structure overview
+
+Repository contains 2 main projects:
+
+* SDK project, which consists of 3 workspaces: `react-native-sdk`, `react-native-sdk-react-navigation`, `react-native-sdk-react-native-navigation`
+* Sample app project (in the `example` folder)
+
+Sample app project exists as dedicated project, because React Native tooling (`Metro` bundler specifically) [doesn't support symbolic links](https://github.com/facebook/metro/issues/1) which is utilized by the [yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) to have a single places for hosting `node_modules`. On the other hand, having symbolic links for SDK modules works fine, because it doesn't use React Native build tooling for running tests and linting.
```

Comment:
```suggestion
The sample app project exists as dedicated project, because the React Native tooling, the `Metro` bundler specifically, [doesn't support symbolic links](https://github.com/facebook/metro/issues/1), and is utilized by the [yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) to have a single places for hosting `node_modules`.
```
I don't think the last sentence is necessary? I may be misunderstanding though, and if so, it should be clarified and not say "on the other hand", because there is nothing on the first hand to begin with.

Reply:
I'm not sure that `and is utilized` is correct replacement here, because it gives the impression that sample app (or React Native tooling?) is utilized by `yarn workspaces`, while it is `symbolic links`.